reference/array/functions/array-diff.xml

```diff
@@ -93,18 +104,54 @@ Array
     </screen>
    </example>
   </para>
+
+  <para>
+   <example>
+    <title><function>array_diff</function> przykład z niepasującymi typami</title>
+    <para>
+     Dwa elementy są uznawane za równe wtedy i tylko wtedy, gdy
+     <literal>(string) $elem1 === (string) $elem2</literal>. To jest,
+     kiedy <link linkend="language.types.string.casting">reprezentacja łańcuchowa</link> jest taka sama.
+    </para>
+    <programlisting role="php">
+     <![CDATA[
+<?php
+// Spowoduje to wygenerowanie Powiadomienia, że tablica nie może zostać rzutowana na ciąg znaków.
```

Comment:
```suggestion
// Spowoduje to wygenerowanie komunikatu E_NOTICE, że tablica nie może zostać rzutowana na ciąg znaków.
```

w przeciwnym razie nie wiedziałbym o jakiego rodzaju "powiadomieniu" jest mowa.  Przekład nie jest dosłowny, ale powinien być o wiele bardziej zrozumiały.

Reply:
Tutaj zamieniam na "Ostrzeżenie", było w oryginale jest "Notice", co może być tłumaczone jako ostrzeżenie lub niezalecane powiadomienie. Plik zrzutu z kompilatora w załączeniu.
![Warning-notice](https://github.com/php/doc-pl/assets/49682955/45140194-539c-427b-b962-d92c72d24c3a)

Reply:
To jest ciekawy przypadek, bo śmiem twierdzić że dokumentacja po angielsku jest po prostu błędna lub raczej nieaktualna, i że mają na myśli właśnie e_notice. W tej sytuacji zawsze powstaje dylemat czy powinniśmy poprawiać takie rzeczy tłumacząc, czy nie. Prawdopodobnie powinniśmy tłumaczyć dosłownie i czekać na poprawkę w pierwowzorze.

Z drugiej strony, może chodzić własnie o to, że użyli "notice" jako wyrazu, który ujmie oba te przypadki, a więcej detali jest opisanych w innych częściach tej strony (czyli różnica między e_notice a e_warning zależnie od wersji PHP)